src/dnx/pal.win32.cpp

```diff
@@ -81,19 +83,65 @@ BOOL GetFullPath(LPCTSTR szPath, LPTSTR pszNormalizedPath)
     return TRUE;
 }
 
+namespace
+{
+    std::wstring get_runtime_path(TraceWriter& trace_writer)
+    {
+        std::vector<wchar_t*> servicing_locations =
+        {
+            L"DNX_SERVICING",
+            L"ProgramFiles(x86)"
+        };
+
+        wchar_t servicing_location_buffer[MAX_PATH];
+        // The servicing index should be always under %ProgramFiles(x86)% however
+        // on 32-bit OSes there is only %ProgramFiles%
+        if (GetEnvironmentVariable(L"ProgramFiles(x86)", servicing_location_buffer, MAX_PATH) == 0)
```

Comment:
Should strings like "ProgramFiles(x86)" etc. be #define's to ensure we don't have syntax mistakes as code changes?